test/functional/api/cas/cache_config.py

```diff
@@ -143,6 +143,21 @@ def __eq__(self, other):
             and self.wake_up_time == other.wake_up_time
         )
 
+    def __str__(self):
+        ret = ["activity threshold: " +
+               (f"{self.activity_threshold} " if self.activity_threshold is not None
+                else "default"),
+               "flush max buffers: " +
+               (f"{self.flush_max_buffers} " if self.flush_max_buffers is not None
+                else "default"),
+               "staleness time: " +
+               (f"{self.staleness_time} " if self.staleness_time is not None
+                else "default"),
+               "wake up time: " +
+               (f"{self.wake_up_time} " if self.wake_up_time is not None
+                else "default")]
```

Comment:
All those "default" strings should have trailing space to fit the desired convention.
Or even better, you can put each parameter like this to save a few bytes of code ;)
`[f"activity threshold: {self.activity_threshold or 'default'} ", ...]`

Reply:
Removed trailing spaces as there is join(" | ") after creating the list